bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

```diff
@@ -61,28 +61,46 @@ default String getId() {
         return null;
     }
 
-    @Nullable
-    @JsonIgnore
     @Override
     default String getFieldType() {
-        // explicitly setting null, since form container does not have a field type, but other containers like panel have a field type
-        return null;
+        return FieldType.FORM.getValue();
     }
 
     @JsonIgnore
     @Override
     default boolean isEnabled() {
-        // explicitly setting true, since form container does not have enabled property, but other containers like panel have a field type
+        // explicitly setting true, since form container does not have enabled property, but other containers like panel have enabled
         return true;
     }
 
+    @Override
+    @JsonIgnore
+    default Label getLabel() {
+        // explicitly setting null, since form container does not have label, but other containers like panel have a label
+        return null;
+    }
+
     @JsonIgnore
     @Override
     default boolean isVisible() {
-        // explicitly setting true, since form container does not have visible property, but other containers like panel have a field type
+        // explicitly setting true, since form container does not have visible property, but other containers like panel have visible
```

Comment:
`are` visible

Reply:
can you approve this I will add comments in the next PR